source/common/amqpClient.js

```diff
@@ -534,6 +534,7 @@ class AmqpCli {
     amqp.connect(this.options).then((conn) => {
       log.debug('Connect OK');
       conn.on('error', this._errorHandler);
+      conn.on('close',this._errorHandler);
```

Comment:
In which scenario, we need handle the 'close' event?

Reply:
rabbitmq server restarted.

Reply:
if not handle, restart mq server means you must restart all the node for cluster, some times mq server may has problem.